### DISCUSSION TITLE
Tasmota sensors - two rf receivers are not supported in the same time

### DISCUSSION THREAD

**Discussion by pawol** [arendst/Tasmota#11890] [Ideas]
### PROBLEM DESCRIPTION
I did RF gateway using tasmota sensors and popular RF receiver 315MHz.
At the beginning I used only one receiver 315 MHz zonnected to port GPIO12.
Tasmota had set to this port RFreceiv - it works well

Last time I have added second RF receiver connected to the port GPIO13.

When I switch RFreceiv between GPIO12 and GPIO13 in the configuration menu of tasmota sensors then receivers works properly
but when I enable both ones at once then only works the an RF receiver connected to the port GPIO12.

Bug:
Cannot use both RF receivers at once despite configuration in tasmota menu

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [ ] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [ ] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [ ] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [ ] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [ ] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [ ] Device used (e.g., Sonoff Basic): _____
- [ ] Tasmota binary firmware version number used: _____
  - [ ] Pre-compiled
  - [ ] Self-compiled
- [ ] Flashing tools used: _____
- [ ] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
  Configuration output here:
21:20:55.052 MQT: stat/tasmota_F7567F/RESULT = {"NAME":"RF receiver","GPIO":[1,1,1,1,1,1,1,1,1152,1152,1,1,1,1],"FLAG":0,"BASE":18}
21:20:55.263 MQT: stat/tasmota_F7567F/RESULT = {"Module":{"18":"Generic"}}
21:20:55.364 MQT: tele/tasmota_F7567F/STATE = {"Time":"2021-04-25T21:20:55","Uptime":"0T00:00:09","UptimeSec":9,"Heap":24,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":22,"MqttCount":1,"Wifi":{"AP":1,"SSId":"Network 4G","BSSId":"40:31:3C:03:A1:35","Channel":12,"RSSI":100,"Signal":-39,"LinkCount":1,"Downtime":"0T00:00:03"}}
21:20:55.520 MQT: stat/tasmota_F7567F/RESULT = {"GPIO0":{"0":"None"},"GPIO1":{"0":"None"},"GPIO2":{"0":"None"},"GPIO3":{"0":"None"},"GPIO4":{"0":"None"},"GPIO5":{"0":"None"},"GPIO9":{"0":"None"},"GPIO10":{"0":"None"},"GPIO12":{"1152":"RFrecv"},"GPIO13":{"1152":"RFrecv"},"GPIO14":{"0":"None"},"GPIO15":{"0":"None"},"GPIO16":{"0":"None"},"GPIO17":{"0":"None"}}


```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
  Rules output here:
21:17:45.188 MQT: stat/tasmota_F7567F/RESULT = {"Rule1":"OFF","Once":"OFF","StopOnError":"OFF","Length":0,"Free":511,"Rules":""}
21:17:45.399 MQT: stat/tasmota_F7567F/RESULT = {"Rule2":"OFF","Once":"OFF","StopOnError":"OFF","Length":0,"Free":511,"Rules":""}
21:17:45.650 MQT: stat/tasmota_F7567F/RESULT = {"Rule3":"OFF","Once":"OFF","StopOnError":"OFF","Length":0,"Free":511,"Rules":""}


```
- [ ] Provide the output of this command: `Status 0`:
21:14:49.605 MQT: stat/tasmota_F7567F/STATUS = {"Status":{"Module":18,"DeviceName":"Odbiornik RF","FriendlyName":["Odbiornik RF"],"Topic":"tasmota_F7567F","ButtonTopic":"0","Power":0,"PowerOnState":0,"LedState":1,"LedMask":"FFFF","SaveData":1,"SaveState":1,"SwitchTopic":"0","SwitchMode":[0,0,0,0,0,0,0,0],"ButtonRetain":0,"SwitchRetain":0,"SensorRetain":0,"PowerRetain":0,"InfoRetain":0,"StateRetain":0}}
21:14:49.611 MQT: stat/tasmota_F7567F/STATUS1 = {"StatusPRM":{"Baudrate":115200,"SerialConfig":"8N1","GroupTopic":"tasmotas","OtaUrl":"http://ota.tasmota.com/tasmota/release/tasmota-sensors.bin.gz","RestartReason":"External System","Uptime":"0T02:27:00","StartupUTC":"2021-04-25T16:47:49","Sleep":50,"CfgHolder":4617,"BootCount":108,"BCResetTime":"2020-05-29T16:34:51","SaveCount":214,"SaveAddress":"F4000"}}
21:14:49.617 MQT: stat/tasmota_F7567F/STATUS2 = {"StatusFWR":{"Version":"9.3.1(sensors)","BuildDateTime":"2021-03-09T16:13:23","Boot":31,"Core":"2_7_4_9","SDK":"2.2.2-dev(38a443e)","CpuFrequency":80,"Hardware":"ESP8266EX","CR":"451/699"}}
21:14:49.621 MQT: stat/tasmota_F7567F/STATUS3 = {"StatusLOG":{"SerialLog":2,"WebLog":2,"MqttLog":0,"SysLog":0,"LogHost":"","LogPort":514,"SSId":["Network 4G",""],"TelePeriod":300,"Resolution":"558180C0","SetOption":["8000A009","2805C8000100060000005A0A000000000000","00008080","00006000","00000000"]}}
21:14:49.633 MQT: stat/tasmota_F7567F/STATUS4 = {"StatusMEM":{"ProgramSize":645,"Free":356,"Heap":20,"ProgramFlashSize":1024,"FlashSize":4096,"FlashChipId":"1640E0","FlashFrequency":40,"FlashMode":3,"Features":["00000809","8FDA8787","0415A005","B7FFBFCF","01DA9BC4","64367CC7","00084052","20000000","00000000"],"Drivers":"1,2,3,4,5,6,7,8,9,10,12,14,16,17,20,21,24,29,34","Sensors":"1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,26,28,31,34,37,39,40,42,43,51,52,55,56,58,59,62,64,66,67,74"}}
21:14:49.643 MQT: stat/tasmota_F7567F/STATUS5 = {"StatusNET":{"Hostname":"tasmota_F7567F-5759","IPAddress":"192.168.2.219","Gateway":"192.168.2.1","Subnetmask":"255.255.255.0","DNSServer":"192.168.2.1","Mac":"5C:CF:7F:F7:56:7F","Webserver":2,"WifiConfig":4,"WifiPower":17.0}}
21:14:49.650 MQT: stat/tasmota_F7567F/STATUS6 = {"StatusMQT":{"MqttHost":"192.168.2.3","MqttPort":1883,"MqttClientMask":"DVES_%06X","MqttClient":"DVES_F7567F","MqttUser":"DVES_USER","MqttCount":2,"MAX_PACKET_SIZE":1200,"KEEPALIVE":30}}
21:14:49.657 MQT: stat/tasmota_F7567F/STATUS7 = {"StatusTIM":{"UTC":"2021-04-25T19:14:49","Local":"2021-04-25T21:14:49","StartDST":"2021-03-28T02:00:00","EndDST":"2021-10-31T03:00:00","Timezone":99,"Sunrise":"06:39","Sunset":"20:56"}}
21:14:49.663 MQT: stat/tasmota_F7567F/STATUS10 = {"StatusSNS":{"Time":"2021-04-25T21:14:49"}}
21:14:49.669 MQT: stat/tasmota_F7567F/STATUS11 = {"StatusSTS":{"Time":"2021-04-25T21:14:49","Uptime":"0T02:27:00","UptimeSec":8820,"Heap":20,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":2,"Wifi":{"AP":1,"SSId":"Network 4G","BSSId":"40:31:3C:03:A1:35","Channel":12,"RSSI":100,"Signal":-41,"LinkCount":1,"Downtime":"0T00:00:03"}}}

```
- [ ] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  Console output here:


```

### TO REPRODUCE
connect two rf receivers into the port D12 and D13 and enable RFreceiv for them

### EXPECTED BEHAVIOUR
Receive data from two receivers parallely

### SCREENSHOTS
_If applicable, add screenshots to help explain your problem._

### ADDITIONAL CONTEXT
_Add any other context about the problem here._

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by sfromis** [comment]
It is not a bug that two of the same type are not supported on the same device. It is just how Tasmota works. It cannot work to assign the same gpio function to two pins, unless the function provides a number-dropdown to the right, like when you config Relay pins.

**Comment by Jason2866** [comment]
This feature was already requested. Theo answered he will not change, you can use a second device for.
